presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -44,6 +44,21 @@ memory intensive queries. It is still possible that the query runner will fail
 to divide intermediate data into chunks small enough that every chunk fits into
 memory, leading to `Out of memory` errors while loading the data from disk.
 
+Disk usage
+----------
+
+Spilling intermediate results to disk and retrieving it back is expensive
```

Comment:
Suggestion: start with the recommendation, then describe why.
For best performance for spill-to-disk, it is recommended to provide multiple paths on separate local devices (property ``spiller-spill-path`` in :ref:`tuning-spilling'). Spilling intermediate results to disk and retrieving them is expensive in terms of IO operators. Thus, queries that use spill will likely become throttled by disk. It may also be beneficial to store spill data on HDFS, provided that it is run on the same machines as the Presto cluster.